src/modules/Common/data-components/Username.tsx

```diff
@@ -74,11 +87,26 @@ const DataUsername = ({
         exportName={exportName}
       />
       <div className="fr-mt-8w">
-        <BotProbabilityGraph
+        <BarGraph
+          title="Bot probability"
+          subtitle="In number of users, views on bar chart"
           data={botRepartition.map((value: number, index: number) => ({
             x: index,
             y: value,
           }))}
+          xAxis={{
+            categories: [...Array(botRepartition.length)].map((_, index) => `${index}%`),
+          }}
+        />
+      </div>
+      <div className="fr-mt-8w">
+        <BarGraph
+          title="Nb of tweets per user"
+          subtitle="In number of users, views on bar chart"
```

Comment:
We need 4 strings configuration for this component :
- title ✅
- subtitle ✅
- y axis legend : ❌ 
- x axis legend : ❌

Reply:
In this case, y axis legend string could be `Nb of users`and x axis legend `Nb of Tweets`

Reply:
No need for xaxis legend as it uses the title of graph for now